pyroomacoustics/bss/tests/test_bss.py

```diff
@@ -16,19 +16,18 @@
             'examples/input_samples/cmu_arctic_us_aew_a0003.wav',
             ],
         ]
+# If True test the bss algorithm
+choices = {'auxIVA' : True, 'ILRMA' : True, 'sparseauxIVA' : True}
 
-def test_auxiva():
+# List of frame lengths to test
+L = [256, 512, 1024, 2048, 4096]
 
-    # STFT frame length
-    L = 256
+def test_bss(algo='auxiva', L=256):
```

Comment:
I see that you have regrouped testing of all algorithms here. This is nice for reduction of code duplication.

The test suite will call all functions that start with `test_` or similar, however, I don't think it will provide argument. Thus, during test, `test_bss()` will be only called once and the other algorithms won't be tested. If you want to run all the tests, you should add short wrapper functions, e.g.,
```
def test_bss_auxiva_256():
    test_bss(algo='auxIVA', L=256)
```
and repeat for all parameters to test.